### DISCUSSION TITLE
Expected time for approval of new watch face?

### DISCUSSION THREAD

**Discussion by orenbenkiki** [zepp-health/zeppos-forum#254] [Developer Feedback]
I created an LCARS style watchface for the Amazfit band 7 and submitted it for review a week ago: https://console.zepp.com/#/service/app/audit/detail/watchface/3262

I haven't received any feedback about it yet. Can anyone say what's the expected time for this?

**Comment by MyoungXUE** [comment]
@orenbenkiki 

Hello, due to China's National Day holiday some time ago, the audit work only returned to normal in the past two days, the current backlog of dials is relatively large, please be patient, we expect to complete the audit work in two weeks!

**Comment by orenbenkiki** [comment]
No problem - I was just wondering what the ETA was.
At any rate, it has now been approved - many thanks!
